go.mod

```diff
@@ -55,3 +55,5 @@ require (
 )
 
 replace k8s.io/apimachinery => ./vendor_fixes/k8s.io/apimachinery
+
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
I'm not sure I understand why we need both this _and_ the ENV variable for the go modules proxy. Shouldn't the env variable alone be sufficient (and a smaller change)? And then we can fix the dependencies when updating opencensus?